source/manual/setting-up-new-rails-app.html.md

```diff
@@ -16,137 +16,174 @@ parent: "/manual.html"
 [release]: https://release.publishing.service.gov.uk/applications
 [deploy-jenkins]: https://deploy.integration.publishing.service.gov.uk/job/Deploy_App/
 [docs-applications]: https://github.com/alphagov/govuk-developer-docs/blob/master/data/applications.yml
+[get-started]: https://docs.publishing.service.gov.uk/manual/get-started.html
+[linting]: https://docs.publishing.service.gov.uk/manual/configure-linting.html
+[rails-conv]: https://docs.publishing.service.gov.uk/manual/conventions-for-rails-applications.html
+[naming]: https://docs.publishing.service.gov.uk/manual/naming.html
+[auto-config]: https://docs.publishing.service.gov.uk/manual/configure-github-repo.html#auto-configuration
+[app-list]: https://docs.publishing.service.gov.uk/#applications
 
-## Familiarise yourself with the conventions
+## Before you start
 
-We have documented [conventions for Rails
-applications](/manual/conventions-for-rails-applications.html) and
-[conventions for naming applications](/manual/naming.html) in GOV.UK.
+Before you create a Rails app, you must complete all steps in the [GOV.UK developer get started documentation](get-started).
 
-## Bootstrapping a new application
+## Create a new Rails app
 
-To create a rails app, run the following (skip uncommon stuff).
+1. To create a Rails app, go to the development space on your local machine and run the following in the command line:
 
-```sh
-rails new myapp --skip-javascript --skip-test --skip-bundle --skip-spring --skip-action-cable --skip-action-mailer --skip-active-storage
-```
+    ```sh
+    rails new myapp --skip-javascript --skip-test --skip-bundle --skip-spring --skip-action-cable --skip-action-mailer --skip-active-storage
+    ```
 
-Replace the Gemfile with the gems you need. Here is an example.
+    The `--skip` commands exclude unneeded Rails plugins.
```

Comment:
I think "Rails components" might be better, they're first-party, not third-party, which is what "plugin" implies to me.